src/csharp/Microsoft.Spark.E2ETest/Hadoop/FileSystemTests.cs

```diff
@@ -52,5 +53,28 @@ public void TestDelete()
 
             Assert.False(Directory.Exists(path));
         }
+
+        [Fact]
+        public void TestExists()
```

Comment:
Could you please add a documentation comment describing the test?

Reply:
Added the documentation about the test